install.js

```diff
@@ -70,6 +70,28 @@ const installStaticFiles = (packageRoot, task) => {
   }
 };
 
+/**
+ * Replace a script inside `package.json` if it doesn't contain the expected default script.
+ * This ensures that users can add script snippets representing custom needs inside these scripts
+ * without having their changes overridden on installation. If the default script is not contained, however,
+ * it is inserted with everything else removed.
+ * This method mutates the `packageJson` argument passed in.
+ */
+const replaceScripts = (scripts, packageJson) => {
+  Object.entries(scripts).forEach(scriptData => {
```

Comment:
Alternatively, consider

```suggestion
  Object.entries(scripts).forEach(([scriptName, scriptContents]) => {
```